src/SwedbankPay.Sdk.Infrastructure/JsonSerialization/CustomRegionInfoConverter.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Globalization;
+using System.Linq;
+
+namespace SwedbankPay.Sdk.JsonSerialization
+{
+    public class CustomRegionInfoConverter
+    {
+        public bool CanConvert(Type objectType)
+        {
+            return objectType == typeof(RegionInfo);
+        }
+
+
+        //public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+        //{

```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)